### PR TITLE
[test/plugin] Print systeminfo on failure

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by dmsolr** [comment]
I think we don't require developers to set any limited optional which likes `network`. In my opinion, we provide this option as `${SCENARIO}-${BUILD_NO}` by default.

File: test/plugin/run.sh [hunk 1]
```diff
@@ -77,12 +77,17 @@ exitWithMessage() {
 exitAndClean() {
     elapsed=$(( `date +%s` - $start_stamp ))
     num_of_testcases="`ls -l ${task_state_house} |grep -c FINISH`"
+    [[ $1 -eq 1 ]] && printSystemInfo
```
Comment (dmsolr) [inline test/plugin/run.sh]: move to L74?

Comment (arugal) [reply test/plugin/run.sh]: ```exitWithMessage``` doesn't seem to be the only entry to ```exitAndClean```.

**Pull request merged**